ts/nestjs/src/database/api.ts

```diff
@@ -0,0 +1,10 @@
+import { api } from 'encore.dev/api';
+
+// We add an `api` call here in order for Encore to recognize the "database" folder
```

Comment:
I don't think this is a good pattern to encourage. What was the motivation for doing it this way?

Reply:
Yes, I agree. Not great. I wanted to put the database in separate service so that I could dependency inject it NestJS-style but in order to have a database I needed to have a `api` call for Encore to recognize it as a service. 

I have now refactored a bit and put the `authHandler` and the database in the same service and renamed it to `core`, then I could remove the dummy endpoint. What do you think?